docs/source/get_started/install.md

```diff
@@ -84,36 +45,41 @@ To exit the environment:
 ```bash
 deactivate
 ```
-</details>
 
-<details>
-<summary><b>Click to expand instructions for <code>pipenv</code></b></summary>
 
-Install `pipenv` as follows:
+### How to create a new virtual environment without using `venv`
+
+ [The second, less popular option is to install Conda.](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) Than from your terminal:
 
 ```bash
-pip install pipenv
+conda create --name kedro-environment python=3.10 -y
 ```
 
-Create a directory for working with your project and navigate to it. For example:
+The example below uses Python 3.10, and creates a virtual environment called `kedro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12) for your project, and you can name it anything you choose.
+
+The `conda` virtual environment is not dependent on your current working directory and can be activated from any directory:
```

Comment:
Is this conda or venv? I see conda instruction under venv section

Reply:
maybe you should better have a look to final version on https://github.com/kedro-org/kedro/blob/make-pip-and-venv-installation-instructions-more-visible/docs/source/get_started/install.md not for a code difference, looks like it's hard to understand here. I see it's all right with sections